registry/consul/backend.go

```diff
@@ -100,6 +122,54 @@ func (b *be) WatchNoRouteHTML() chan string {
 	return html
 }
 
+func (b *be) UpdateAliases(config string) error {
```

Comment:
Move the parsing code into the `route/parse_new.go` since there you already have all the parse helpers. Return a list of alias names to register, e.g.

```go
func ParseAliases(config string) (names []string, error) {...}
```